esp/esp/db/forms.py

```diff
@@ -164,6 +164,9 @@ def __init__(self, field_name='', field=None, key_type=None, to_field=None,
                  error_messages=None, show_hidden_initial=False, shadow_field_name=None,
                  *args, **kwargs):
 
+        error_css_class = 'error'
+        #required_css_class = 'required' # in case someone wants to make changes later
```

Comment:
Huh?  What are these?

Reply:
I got those from https://docs.djangoproject.com/en/1.8/ref/forms/api/#styling-required-or-erroneous-form-rows, under the impression that it was necessary for `.form_error` in the CSS, but correct me on that if it's wrong.

Reply:
Oh, ok!  Did you test that the changes actually look right with our CSS?  We do a lot of special stuff.

Also, no need to include the commented line -- if you want you can instead link to the docs which give a lot more info.

Reply:
Yes, you asked about the CSS below, but the end result looked right when I tested it. Okay -- I'll change the commented line.